### PR TITLE
fix: add missing pmie webhook action configuration functionality

### PR DESCRIPTION
**Pull request by natoscott**
Resolves Red Hat issue RHEL-13760

### CODE REVIEW

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
[citest]

File: tests/check_notification.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: MIT
+---
+# yamllint disable rule:line-length
+- name: Check if webhook notification is set
+  command: grep -e 'webhook_endpoint .* {{ __test_webhook }}' /var/lib/pcp/config/pmie/config.default
```
Comment (richm) [inline tests/check_notification.yml:5]: This doesn't match.  Are you trying to match something like
```
  & shell 10 min "pmie_webhook 'https://example.com:4242/endpoint|Restart unresponsive PMDAs" "|%h|pmda%i[%v]@%h" "'";
```
or something like
```
// 0 global webhook_endpoint = "https://example.com:4242/endpoint"
```
if the latter, then you'll have to use
```
  command: grep 'webhook_endpoint .* "{{ __test_webhook }}"' /var/lib/pcp/config/pmie/config.default
```
to match the double quotes.  If the double quotes are optional, then maybe something like
```
  command: grep -E 'webhook_endpoint .* "?{{ __test_webhook }}"?' /var/lib/pcp/config/pmie/config.default
```
(note that `-E` turns on extended regex, not `-e`)

**Comment by richm** [comment]
[citest]

**Comment by richm** [comment]
@natoscott another issue - grafana is core dumping e.g. rhel9.4 tests_bz1855544.yml - the error is
```
Dec 05 13:21:53 localhost grafana-server[56716]: panic: coremodel framework loaded cue.Value has err: import failed reading dir /pkg/framework/coremodel: readDir: lstat //efi: permission denied
```

**Comment by natoscott** [comment]
Superceded by #181

**Pull request closed without merging**